ee/dtr/user/manage-images/sign-images/trust-with-remote-ucp.md

```diff
@@ -0,0 +1,247 @@
+Signing and pushing trust metadata
+Enter passphrase for cluster2admin key with ID a453196:
+Successfully signed dtr.example.com/admin/trustdemo:1
+```
+
+Within the DTR web interface, you should now be able to see your newly pushed tag with the **Signed** text next to the size. 
+
+![](../../../images/remoteucp-signedimage.png){: .with-border}
+
+
+You could sign this image multiple times if required, whether it's multiple
+teams from the same cluster wanting to sign the image, or you integrating DTR with more remote UCPs so users from clusters 1,
+2, 3, or more can all sign the same image. 
+
+## Enforce Signed Image Tags on the Remote UCP 
+
+We can now enable **Only Run Signed Images** on the remote UCP. To do this,
+login to cluster 2's UCP web interface as an admin. Select **Admin > Admin Settings > Docker Content
+Trust**. 
+
+See [Run only the images you trust](/ee/ucp/admin/configure/run-only-the-images-you-trust/) for more information on only running signed images in UCP.
+
+
+![](../../../images/remoteucp-enablesigning.png){: .with-border}
+
+
+Finally we can now deploy a workload on cluster 2, using a signed
+image from a DTR running on cluster 1. This workload could be a simple `$ docker
+run`, a Swarm Service, or a Kubernetes workload. As a simple test, source a
+client bundle, and try running one of your signed images. 
+
+```
+$ source env.sh
+
+$ docker service create dtr.example.com/admin/trustdemo:1
+nqsph0n6lv9uzod4lapx0gwok
+overall progress: 1 out of 1 tasks
+1/1: running   [==================================================>]
+verify: Service converged
+
+$ docker service ls
+ID                  NAME                    MODE                REPLICAS            IMAGE                                   PORTS
+nqsph0n6lv9u        laughing_lamarr         replicated          1/1                 dtr.example.com/admin/trustdemo:1
+```
+
+## Troubleshooting
+
+If the image is stored in a private repository within DTR, you need to pass credentials to the
+Orchestrator as there is no SSO between cluster 2 and DTR. See the relevant
+[Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-in-the-cluster-that-holds-your-authorization-token) or [Docker
+   Swarm](https://docs.docker.com/engine/swarm/services/#create-a-service-using-an-image-on-a-private-registry)
+   documentation for more details.
+
+### Example Errors
+
+```
+image or trust data does not exist for dtr.example.com/admin/trustdemo:1
+```
+
+This means something went wrong when initiating the repository or signing the
+image, as the tag contains no signing data. 
+
+```
+Error response from daemon: image did not meet required signing policy
+
+dtr.example.com/admin/trustdemo:1: image did not meet required signing policy 
+```
+
+This means that the image was signed correctly, however the user who signed the
+image does not mean the signing policy in cluster 2. This could be because you
```

Comment:
@ollypom "does not mean the signing policy" isn't very clear to me. Can you please expound?

Reply:
Sorry, this should say "however the user who signed the image does not meet the signing policy in cluster 2". 

'meet' not 'mean'. My bad :(